package.json

```diff
@@ -37,28 +37,25 @@
     "lint": "eslint --max-warnings 0 .",
     "lint:format": "prettier --check .",
     "start": "pnpm run build -- --watch",
-    "test": "jest --forceExit --detectOpenHandles --testLocationInResults --json --outputFile=report.json",
-    "test:watch": "pnpm run test -- --watch",
+    "test": "vitest run --coverage",
+    "test:watch": "vitest",
     "typecheck": "tsc --noEmit",
     "release": "semantic-release"
   },
   "devDependencies": {
     "@geprog/eslint-config": "1.0.5",
     "@geprog/semantic-release-config": "1.0.0",
-    "@jest/types": "27.5.1",
-    "@types/jest": "27.4.1",
-    "@types/node": "17.0.23",
     "@types/supertest": "2.0.12",
+    "c8": "7.11.2",
     "eslint": "8.11.0",
-    "jest": "27.5.1",
+    "jsdom": "19.0.0",
     "prettier": "2.6.1",
     "semantic-release": "19.0.2",
     "supertest": "6.2.2",
-    "ts-jest": "27.1.4",
-    "ts-node": "10.7.0",
     "tsup": "5.12.1",
     "typescript": "4.6.3",
-    "vite": "2.8.6"
+    "vite": "2.9.8",
+    "vitest": "0.10.5"
```

Comment:
Could you add https://www.npmjs.com/package/@vitest/ui.

Reply:
@anbraten I would prefer not to add this to the project. After all there is https://vitest.dev/guide/ide.html